packages/lodestar/src/chain/chain.ts

```diff
@@ -204,7 +203,7 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
       fork => this.config.types.Version.equals(currentVersion, intToBytes(fork.previousVersion, 4)));
     return {
       forkDigest: this.currentForkDigest,
-      nextForkVersion: nextVersion? intToBytes(nextVersion.currentVersion, 4) : MAX_VERSION,
+      nextForkVersion: nextVersion? intToBytes(nextVersion.currentVersion, 4) : currentVersion.valueOf() as Uint8Array,
```

Comment:
this is to address spec: `If no future fork is planned, set next_fork_version = current_fork_version to signal this fact`